### DISCUSSION TITLE
Welcome to WprimeSearch Discussions!

### DISCUSSION THREAD

**Discussion by vargasa** [vargasa/WprimeSearch#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rishabhCMS** [comment]
do you believe in parallel universes?
I mean, you are a grad student in one universe trying to making it big but in the parallel universe you are just acing through life and thriving

**Reply by vargasa** [reply]
Even if I did. It doesn't solve any problem
